### PR TITLE
Add patch for jaxlib 0.1.71 in jax 0.2.20 easyconfig to fix environment during build

### PR DESCRIPTION
**Pull request by Flamefire**
Basically applies my proposed change from https://reviews.llvm.org/D109873

edit: for https://github.com/easybuilders/easybuild-easyconfigs/pull/14003

### CODE REVIEW

File: easybuild/easyconfigs/j/jax/jaxlib-0.1.71_fix-llvm-env.patch [hunk 1]
```diff
@@ -0,0 +1,28 @@
+diff --git a/third_party/llvm/workspace.bzl b/third_party/llvm/workspace.bzl
```
Comment (boegel) [inline easybuild/easyconfigs/j/jax/jaxlib-0.1.71_fix-llvm-env.patch:6]: @Flamefire Please add a comment at top of the patch to explain what it does, why it's needed, where it comes from (or author)

**Review by boegel — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/j/jax/jaxlib-0.1.71_fix-llvm-env.patch (hunk 1 above)
Comment (Flamefire) [reply easybuild/easyconfigs/j/jax/jaxlib-0.1.71_fix-llvm-env.patch:6]: Done

**Review by boegel — CHANGES_REQUESTED** [review]
@Flamefire SHA256 checksum for the patch is missing (tests should catch that)

**Comment by Flamefire** [comment]
Done

**Review by boegel — APPROVED** [review]

**Pull request merged**